guides/common/modules/con_switching-between-hosts.adoc

```diff
@@ -0,0 +1,7 @@
+[id="switching-between-hosts_{context}"]
+= Switching between Hosts
+
+When you are on a particular host in the {ProjectWebUI}, you can navigate between hosts without leaving the page by using the host switcher.
+Click *⇄* next to the hostname.
+
+A list of hosts is displayed in alphabetical order with a pagination arrow and a search bar to find the host you are looking for.
```

Comment:
```suggestion
This displays a list of hosts in alphabetical order with a pagination arrow and a search bar to find the host you are looking for.
```
I don't think that this one sentence should be standalone. If you have a reason for this please say.

Reply:
Let's keep yours. It will give better coherence.